.github/workflows/bundle-size.yml

```diff
@@ -0,0 +1,21 @@
+name: Compressed Size
+
+on:
+    pull_request:
+    workflow_dispatch:
+
+jobs:
+    build:
+        name: Bundle size check
+        runs-on: ubuntu-latest
+
+        steps:
+            - uses: actions/checkout@v2
+            - name: Use desired version of NodeJS
+              uses: actions/setup-node@v2
+              with:
+                  node-version: '12'
+                  cache: npm
+            - uses: preactjs/compressed-size-action@v2
+              with:
+                  pattern: '{release/**/*.js,release/**/*.tsx,release/**/*.css,release/**/*.php}'
```

Comment:
@fashuvo Do we want to check size of PHP files here? JS, TS & CSS seems fine.

Reply:
I wasn't sure about that as well, but I thought we track size changes of PHP files too.

Why shouldn't we check the PHP file size? Is that because we don't work with PHP that often, or is there any other reason?

Reply:
> Why shouldn't we check the PHP file size? 

I don't think we use much external lib/helpers with PHP, so a size increase that way is less likely to happen. Usually, bundle size checks are required for front-end assets like JS, CSS, Images etc. PHP files are called and executed only when required.

@RadoslavGeorgiev what do you think about checking bundle size for PHP files?

Reply:
I agree with @achyuthajoy's suggestion. As this check targets the client-side bundle generated by webpack, which does not include PHP files, it's safe to exclude what is not considered an asset file. In fact, it may be reasonable to include only `release/assets` and `release/dist` in this check.

Through this check, we'd like to keep the total size of files (assets) downloaded by the browser under reasonable limits so that a user's browsing experience (especially on slow devices such as mobiles) is not hampered because of unnecessary files.

`.tsx` files normally get compiled as .`js` files. I wonder if we need them here.

Reply:
Thanks for your detailed explanation @anu-rock 🙇 

Your explanation makes sense, and I have removed checks for PHP and .tsx files.